WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteSourceManager.kt

```diff
@@ -42,22 +45,29 @@ class MySiteSourceManager @Inject constructor(
             cardsSource
     )
 
+    private val showDashboardCards: Boolean
+        get() = mySiteDashboardPhase2FeatureConfig.isEnabled() &&
+                !buildConfigWrapper.isJetpackApp &&
+                selectedSiteRepository.getSelectedSite()?.isUsingWpComRestApi == true
+
+    private val allSupportedMySiteSources: List<MySiteSource<*>>
```

Comment:
I'm not sure this name is justified here. Let me know what you think. I used this name as it includes all sources including site independent ones minus any unsupported sources like the dashboard card source.

Reply:
> I'm not sure this name is justified here

I think the name `allSupportedMySiteSources` reflects the sources needed for site in context. We can add/remove from it for any card that comes in the future. To me it's clearer than  `filteredSiteDependentSources`, so 👍